src/neo/ProtocolSettings.cs

```diff
@@ -61,18 +63,54 @@ private ProtocolSettings(IConfigurationSection section)
                 };
             IConfigurationSection section_sl = section.GetSection("SeedList");
             if (section_sl.Exists())
-                this.SeedList = section_sl.GetChildren().Select(p => p.Get<string>()).ToArray();
+                this.SeedList = section_sl.GetChildren()
+                    .AsParallel()
+                    .Select(p => GetIpEndPoint(p.Get<string>()))
+                    .Where(u => u != null)
+                    .ToArray();
             else
                 this.SeedList = new[]
                 {
-                    "seed1.neo.org:10333",
-                    "seed2.neo.org:10333",
-                    "seed3.neo.org:10333",
-                    "seed4.neo.org:10333",
-                    "seed5.neo.org:10333"
+                    GetIpEndPoint("seed1.neo.org:10333"),
+                    GetIpEndPoint("seed2.neo.org:10333"),
+                    GetIpEndPoint("seed3.neo.org:10333"),
+                    GetIpEndPoint("seed4.neo.org:10333"),
+                    GetIpEndPoint("seed5.neo.org:10333")
```

Comment:
This may block the initialization. Moreover, these DNS queries can be performed in parallel.